bccsp/idemix/bridge/bridge_test.go

```diff
@@ -764,9 +764,9 @@ var _ = Describe("Idemix Bridge", func() {
 					IssuerPublicKey,
 					credRequest,
 					nil,
-					&bccsp.IdemixCredentialRequestSignerOpts{IssuerNonce: []byte("pine-aple-pine-apple-pinapple")},
+					&bccsp.IdemixCredentialRequestSignerOpts{IssuerNonce: []byte("pin-apple-pine-apple-pineapple")},
```

Comment:
Did you mean to spell `pin` instead of `pine` in the first instance

Reply:
It just needed to be a word and the total length less than 32 characters.